src/admin-views/settings/tickets-commerce/stripe/connect/active/connection.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * The Template for displaying the Tickets Commerce PayPal connection details.
+ *
+ * @version 5.1.10
+ *
+ * @since 5.1.10
+ *
+ * @var Tribe__Tickets__Admin__Views                  $this               [Global] Template object.
+ * @var string                                        $plugin_url         [Global] The plugin URL.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Merchant $merchant           [Global] The merchant class.
+ * @var TEC\Tickets\Commerce\Gateways\PayPal\Signup   $signup             [Global] The Signup class.
```

Comment:
```suggestion
 * The Template for displaying the Tickets Commerce Stripe connection details.
 *
 * @version TBD
 *
 * @since TBD
 *
 * @var Tribe__Tickets__Admin__Views                  $this               [Global] Template object.
 * @var string                                        $plugin_url         [Global] The plugin URL.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Merchant $merchant           [Global] The merchant class.
 * @var TEC\Tickets\Commerce\Gateways\Stripe\Signup   $signup             [Global] The Signup class.
```